docs/source/concepts/DynamicalStructureFactorFromAbInitio.rst

```diff
@@ -8,25 +8,30 @@ Introduction
-describe the general idea how theoretical S from ab-initio is calculated by available plugins in Mantid.
+describe in general how theoretical S is calculated from from *ab initio* data by plugins in Mantid.
 
-During an inelastic neutron scattering experiment sample is exposed to neutron flux and response of a sample in the
-form of dynamical structure factor  is recorded. In principle, one obtains phonon (vibrational) spectrum which is often
-quite  difficult to analyse. In order to better understand experimental results one can use modeling methods. One of the
-most successful theoretical method in predicting phonon spectrum is ab-initio method DFT [1]_.
+During an inelastic neutron scattering experiment, a sample is exposed to neutron flux and a response is recorded in the form of dynamical structure factor, :math:`S(\mathbf{Q}, \omega)`.
+In principle, one obtains a vibrational spectrum which can be quite difficult to analyse; in crystalline materials this
+related to the wavevector-dependent *phonon* spectrum.
+In order to better understand experimental outputs, one can compare with results from modelling.
+*Ab initio* calculations, especially within density-functional theory (DFT) [#Kohn1964]_, have proven quite successful in predicting vibrational spectra.
 
 .. image:: ../images/dft_phonon_scheme.png
     :align: center
 
-Usual approach in calculating phonon spectra within DFT is presented in the figure above. First, one defines initial
-guess for the structure in interest. The initial guess should be as close as possible to an experimental structure.
-Then, one optimise structure within DFT. By optimising structure one should understand finding the closest to the
-initial guess local minimum of energy within accuracy of DFT method. At the closest minimum there
-should be no effective force on atoms. For that structure the dynamical matrix, e.g, second derivative of system Hamiltonian with respect to
-atomic displacements is calculated. Eigenvectors obtained from diagonalisation of this matrix are atomic displacements
-and eigenvalues are squared frequencies of vibrations. Those vibrations are sometime called *modes* or *fundamentals*.
-Using atomic displacements and frequencies obtained by DFT method one can calculate theoretical :math:`S(\mathbf{Q}, \omega)`. :math:`S(\mathbf{Q}, \omega)`
-is calculated for each atom separately. Total S is sum over all partial atomic contributions.
+The usual workfow for calculating phonon spectra within DFT is presented in the figure above. First, one defines an
+initial guess for the structure in interest.
+The initial guess should be as close as possible to an experimental structure, and is usually derived from elastic X-ray and/or neutron scattering measurements.
+Then the structure parameters are locally optimised within DFT, finding the nearest structure that minimises the DFT energy.
+At this point, there should be no effective force on the atoms.
```

Comment:
```suggestion
At this point, there should be no effective net force on the atoms.
```

Reply:
It's *ab initio*, all forces should be "net"? We don't get a pairwise breakdown.

Reply:
I was thinking in terms of the atomic potential and Hellman-Fynaman etc. I just added the word net to make it explicit as a non-expert may think it means zero forces, which is different to no net force.

Reply:
Ah, I see what you mean. I find "net" and "effective" to be redundant: I think I prefer `At this point, there should be no net force on the atoms.` which seems more precise.